ChatterboxBackend/Chatterbox/Controllers/UserController.cs

```diff
@@ -0,0 +1,79 @@
+﻿using Chatterbox.Models;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Chatterbox.Controllers
+{
+    [Route("api/[controller]")]
+    public class UserController:Controller
+    {
+        private readonly ChatterContext _context;
+
+        public UserController(ChatterContext context)
+        {
+            _context = context;
+        }
+
+        [HttpPost("login")]
+        public IActionResult LogMeIn([FromBody] LoginDetails details)
+        {
+            Console.WriteLine("NEW LOG",details.Email);
+            var emailCheck = _context.Users.FirstOrDefault(t => t.Email == details.Email);
+
+
+            if (emailCheck == null)
+            {
+                return new ObjectResult(new { status = "failed",message="No such user", user = "" });
+            }
+
+            var items = _context.Users.FirstOrDefault(t => t.Email == details.Email && t.Password == details.Password);
+
+
+            if (items == null)
+            {
+                return new ObjectResult(new { status = "failed",message="Incorrect password", user = "" });
+            }
+            var userDeets = new
+            {
+                id = items.Id,
+                name=items.Name,
+                email=items.Email,
+                image=items.Image
+            };
+            
+            return new ObjectResult(new {status="successful",user=userDeets});
+        }
+        [HttpPost("register/stepone")]
+        public IActionResult RegisterStepOne([FromBody] RegisterDetails details)
+        {
+            var items = _context.Users.FirstOrDefault(t => t.Email == details.Email);

```

Comment:
Rather use `IQueryable.Any()` to check for existence if you're not going to use `items` for anything else.

https://msdn.microsoft.com/en-us/library/bb534338(v=vs.110).aspx